tests/testcontainers-env/src/main/java/org/alfresco/tas/AlfrescoStackInitializer.java

```diff
@@ -298,6 +303,16 @@ private MariaDBContainer createMariaDBContainer()
 
     }
 
+    private JdbcDatabaseContainer createMsSqlContainer()
+    {
+        return new MSSQLServerContainer<>(getImagesConfig().getMsSqlImage())
+                .acceptLicense()
```

Comment:
If you decide to move usage of some values to enum (as above) then you could use them here, like DatabaseType.MSSQL.getPassword()